clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/TitusRegion.java

```diff
@@ -69,6 +73,16 @@ public TitusRegion(
     } else {
       this.featureFlags = featureFlags;
     }
+    if (eurekaName == null) {
+      this.eurekaName = null;
+    } else {
+      this.eurekaName = eurekaName;
+    }
+    if (eurekaRegion == null) {
+      this.eurekaRegion = null;
+    } else {
+      this.eurekaRegion = eurekaRegion;
+    }
```

Comment:
What's the reasoning here. It looks like this should just be set, right? Am i missing anything?

Reply:
```suggestion
this.eurekaName = eurekaName;
this.eurekaRegion = eurekaRegion;
```

Reply:
fixed this , i was setting defaults and was handling it this way , not required actually